src/main/java/mate/controller/DeleteManufacturerController.java

```diff
@@ -0,0 +1,22 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/manufacturers/delete")
+public class DeleteManufacturerController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final ManufacturerService manufacturerService =
+            (ManufacturerService) injector.getInstance(ManufacturerService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+        manufacturerService.delete(Long.valueOf(req.getParameter("id")));
+        resp.sendRedirect("/manufacturers/all");
```

Comment:
same

Reply:
yeap. when u delete and that redirecting to ALL manufacturers) and again and again